### PR TITLE
refactor: convert extern to import

### PR DESCRIPTION
**Pull request by WalterBright**
Pretty simple. This is now possible because the files are now in D rather than C++.

### CODE REVIEW

File: src/toobj.d [hunk 1]
```diff
@@ -64,30 +68,9 @@ import ddmd.backend.type;
...
-void genTypeInfo(Type t, Scope *sc);
-bool isSpeculativeType(Type t);
-
-void toDebug(EnumDeclaration ed);
-void toDebug(StructDeclaration sd);
-void toDebug(ClassDeclaration cd);
-
-void objc_Module_genmoduleinfo_classes();
+alias toSymbol = ddmd.tocsym.toSymbol;
+alias toSymbol = ddmd.glue.toSymbol;
```
Comment (UplinkCoder) [inline src/toobj.d]: How does this work ?

**Comment by yebblies** [comment]
This is beautiful!  If only it compiled...

File: src/toobj.d (hunk 1 above)
Comment (WalterBright) [inline src/toobj.d]: It overloads the toSymbol function(s) in tocsym and glue together.

**Comment by yebblies** [comment]
Feel free to merge when it passes the tester.

**Pull request merged**